src/lib/components/BodyParser/utils.ts

```diff
@@ -44,6 +44,7 @@ export const regExp = {
   symbols: /^[a-zA-Z0-9 ,.";():&%£’€$#@_=/\-'!*?\\/s]{2,}$/,
   image: /.*\.(gif|jpe?g|bmp|png)$/,
   holdexLink: /^(http:\/\/|https:\/\/)(holdex\.io)/,
+  internalLink: /^(\/|#).*/,
```

Comment:
```suggestion
  internalLink: /^(?:\/|#).*/,
```

do you need a capturing group? I don't think so.